lib/daru/io/exporters/excel.rb

```diff
@@ -23,34 +39,96 @@ class Excel < Base
         #
         #   Daru::IO::Exporters::Excel.new(df, "dataframe_df.xls").call
         #
-        # @todo The +opts+ parameter isn't used while creating the Excel Spreadsheet
-        #   yet. Implementing this feature will greatly allow the user to generate a
-        #   Spreadsheet of their choice.
+        # @example Writing to an Excel file with options
+        #   df = Daru::DataFrame.new([[1,2],[3,4]], order: [:a, :b])
+        #
+        #   #=> #<Daru::DataFrame(2x2)>
+        #   #      a   b
+        #   #  0   1   3
+        #   #  1   2   4
+        #
+        #   Daru::IO::Exporters::Excel.new(df,
+        #     "dataframe_df.xls",
+        #     formatting: {
+        #       header: { color: :red, weight: :bold },
+        #       index:  { color: :green },
+        #       data:   { color: :blue }
+        #     },
+        #     display: { index: false }
```

Comment:
...for example, this example is weird, first set formatting of index, then tell we don't want to show it ;)